src/main/resources/application.properties

```diff
@@ -6,4 +6,7 @@ spring.jpa.defer-datasource-initialization=true
 spring.sql.init.mode=always
 spring.sql.init.schema-locations=classpath:schema.sql
 spring.sql.init.data-locations=classpath:data.sql
+security.jwt.token.secret-key=your-very-secret-key-super-super-long-special-key
+security.jwt.token.expire-length=3600000
+
```

Comment:
seconds? minutes? hours? days? months?
we can rename with time unit or append comment 😉